### DISCUSSION TITLE
Running on 64-bit machine

### DISCUSSION THREAD

**Discussion by TalHarel1998** [hjlarry/flask-shop#97] [Q&A] [answered]
Hi!
I can't run the $ flask createdb command and I'm getting this error:
File "c:\users\tal20\onedrive\desktop\tal\tau\2023a\e-commerce\flask-shop\.venv\lib\site-packages\bcrypt\__init__.py", line 33, in <module>
    from . import _bcrypt  # noqa: I100
ImportError: DLL load failed: %1 is not a valid Win32 application.

I've followed all the steps, do you know what to do? I am using 64-bit machine

**Comment by hjlarry** [comment, marked as answer]
It seems the bcrypt install error

https://github.com/pyca/bcrypt/issues?q=+DLL+load+failed
hope this can help you
